CONTRIBUTING.md

```diff
@@ -92,13 +92,17 @@ Use an editor to make your changes and integrate them through github. Read more
 
 # Files
 
-## File Encoding
+## File encoding
 Make sure your text editor preserves file encodings.  
-New files need to have the same encoding as their english counterparts (eg. labsmail1_czech.txt needs to be UTF-8 encoded just like labsmail1_english.txt). If you use Notepad++ you can see the file's encoding in the menu Encoding. vscode shows the files encoding in its status bar.
+New files need to have the same encoding as their english counterparts (eg. labsmail1_czech.txt needs to be UTF-8 encoded just like labsmail1_english.txt). If you use Notepad++ you can see the file's encoding in the menu Encoding. Visual Studio Code shows the files encoding in its status bar.
+
+## Special characters and placeholders
+Quotation marks inside your translations need to be escaped like this `\"Space Station\"`.
+If there are words between `%percent_signs%`, leave them as-is (these are dynamically replaced with numbers in the game later).
 
 ## File type
 ### Achievements
-These files are automatically created based on the `resource/reactivedrop_*.txt` files. You can manually trigger an update of the files through executing `utils/translation-sync-tool.exe`.
+These files are automatically created based on the `resource/reactivedrop_*.txt` files, after you've made an commit. You can manually trigger an update of the files locally through executing `utils/translation-sync-tool.exe` before your commit.
```

Comment:
The command to do the local update is `utils\translation-sync-tool.exe -render` but I'd prefer that translators not do it locally because having multiple languages change in the item schema at once is what causes merge conflict.